fe/fe-core/src/main/java/com/starrocks/http/rest/v2/vo/PartitionInfoView.java

```diff
@@ -177,8 +181,12 @@ public static PartitionView createFrom(PartitionInfo partitionInfo, Partition pa
                     if (lakeTabletOptional.isPresent()) {
                         LakeTablet lakeTablet = lakeTabletOptional.get();
                         try {
+                            WarehouseManager manager = GlobalStateMgr.getCurrentState().getWarehouseMgr();
```

Comment:
what's the code path to reach here? how to identify it should be the background warehouse?

Reply:
Good question. When I debug the code, I clear the Nodes in the Default Warehouse and modify the corresponding positions according to the error log. Next, I will organize the code path and explain why it needs to be set to a background warehouse.

Reply:
In TablePartitionAction, we define an endpoint, /api/v2/catalogs/{%s}/databases/{%s}/tables/{%s}/partition, through which users can obtain detailed information about a Partition.

While traversing a Partition, createFrom converts the Partition's information into a PartitionView. The storagePath field of PartitionView requires a call to getShardInfo to obtain. It use the first tablet to get the related shardInfo.

For such small tasks, placing them in the background warehouse is appropriate.

Reply:
maybe not. check about the partition info, if it contains info about the tablet location. If use background warehouse, the replica location will be the nodes in background warehouse, not the nodes in executing warehouse.